uyuni-tools/reactivate_proxy_clients.py

```diff
@@ -58,13 +58,18 @@
 
     client_mapping = suma_rpc.system.getMinionIdMap(key)
     suma_salt = salt.client.LocalClient()
-    res = suma_salt.cmd_iter("master:{}".format(args.proxy_fqdn), "grains.item", ["id"], tgt_type="grain", timeout = 2)
+    res = suma_salt.cmd_iter("master:{}".format(args.proxy_fqdn), "grains.item", ["id", "saltpath"], tgt_type="grain", timeout = 2)
     for c in res:
         nclients += 1
         if maxclients < nclients:
             break
+        
+        salt_ret = list(c.values())[0]["ret"]
+        c_saltid = salt_ret["id"]
+        c_configpath = "/etc/venv-salt-minion/minion.d/susemanager.conf" \
+            if "venv" in salt_ret.get("saltpath", "") \
+            else "/etc/salt/minion.d/susemanager.conf"
```

Comment:
Here you need expanded if so you can set both client configpath and also client service name (`salt-minion` vs. `venv-salt-minion`)